solutions/src/main/java/ru/sberbank/school/task10/ScalableThreadPool.java

```diff
@@ -1,80 +1,23 @@
 package ru.sberbank.school.task10;
 
-import java.util.ArrayList;
-import java.util.LinkedList;
-import java.util.List;
 import java.util.Objects;
-import java.util.Queue;
-import java.util.concurrent.Callable;
-import java.util.concurrent.Future;
-import java.util.concurrent.FutureTask;
-
-public class ScalableThreadPool implements ThreadPool {
-    private final List<Thread> threadPool;
-    private final Queue<Runnable> runnableQueue = new LinkedList<>();
-    private int threadUniqueNumber;
-    private final int minPoolSize;
+public class ScalableThreadPool extends FixedThreadPool {
```

Comment:
&#39;CLASS_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1744108489>*